src/castTo/castTo.ts

```diff
@@ -0,0 +1,52 @@
+import { CastMapping } from '../@types/cast-mapping';
+import { chain } from '../chain';
+import { entries } from '../entries';
+import { filter } from '../filter';
+import { reduce } from '../reduce';
+
+/**
+ * @module object=>object
+ */
+/**
+ * Cast properties of an object or an array of object, given provided mapping
+ * @param mapping Mapping between key and caster.
+ * @return The function to apply on the object or the array to do cast its properties into another object or array
+ * @example
+ * ```
+ * castTo({a: 1, b: 2, c:3})({a: x => x + x, b: x => x}) // {a: 2, b: 2, c:3}
```

Comment:
Replace this example by the more meanfull one:
```typescript
castTo(
  ({ a }) => parseInt(a, 10), 
  ({ b }) => Math.floor(b), 
  ({ c }) => parseInt(c, 16)
)({ a: '1', b: 2.1, c: 'A' }) // { a: 1, b: 2, c: 10 }'`

Reply:
Done